plugins/catalog-react/src/hooks/useEntityTypeFilter.tsx

```diff
@@ -86,10 +86,11 @@ export function useEntityTypeFilter(): EntityTypeReturn {
     const countByType = entities.reduce((acc, entity) => {
       if (typeof entity.spec?.type !== 'string') return acc;
 
-      if (!acc[entity.spec.type]) {
-        acc[entity.spec.type] = 0;
+      const entityType = entity.spec.type.toLowerCase();
```

Comment:
Should use [toLocaleLowerCase('en-US')](https://github.com/backstage/backstage/pull/4941); I went looking for the ADR but I don't think it ever got created 😞